src/libs/ajax/DatasetBuilder.ts

```diff
@@ -155,6 +246,8 @@ export const DatasetBuilder = (): DatasetBuilderContract => ({
       .details([datasetIncludeTypes.SNAPSHOT_BUILDER_SETTINGS, datasetIncludeTypes.PROPERTIES]);
   },
   getConcepts: (parent: Concept) => Promise.resolve(getDummyConcepts(parent)),
-  requestAccess: (_request) => Promise.resolve(),
+  requestAccess: async (datasetId, _request) => {
+    return await Ajax().DataRepo.dataset(datasetId).createDatasetRequest(convertDatasetAccessRequest(_request));
```

Comment:
I think these should also remove the `_`s
```suggestion
  requestAccess: async (datasetId, request) => {
    return await Ajax().DataRepo.dataset(datasetId).createDatasetRequest(convertDatasetAccessRequest(request));
```